cardano-config/src/Cardano/Config/Presets.hs

```diff
@@ -35,6 +39,12 @@ mainnetConfiguration =
     , pccLogConfig = pure "./configuration/log-configuration.yaml"
     , pccDBPath = pure "./db/"
     , pccApplicationLockFile = pure ""
```

Comment:
Comments on sensible defaults? I would actually like to further separate the `cardano-node` and `cardano-cli`'s configurations in a follow up PR. Essentially the same POM design pattern in each separately. I don't see why they have to share the same configuration. @deepfire

Reply:
I'd say that it's hard to reason about this generally.

Let's decide on a case-by-basis -- sometimes sharing makes sense, sometimes it doesn't.

Reply:
I strongly disagree with sharing configuration types between `cardano-cli` and `cardano-node`.  I believe @erikd made a comment expressing a similar view. Having simpler cli parsers gives each cli more flexibility and less coupling.

Reply:
@Jimbo4350, so you say we don't want to share `CardanoConfiguration`, right?

Reply:
@deepfire Exactly.